src/tasks.py

```diff
@@ -63,30 +67,48 @@ def command(
     """
     input_files = get_input_files(pipe_result, input_files or [])
     output_files = []
-    base_command = ["<REPLACE_WITH_COMMAND>"]
-    base_command_string = " ".join(base_command)
 
-    for input_file in input_files:
+    task_report = Report("SSH log analyzer report")
+
+    try:
+        log_year = int(task_config.get('log_year'))
+    except (TypeError, ValueError):
+        log_year = None 
+
+    ssh_analysis_task = LinuxSSHAnalysisTask(log_year=log_year)
+    analyzer_output_priority = Priority.LOW
+
+    df = ssh_analysis_task.read_logs(input_files=input_files)
+    if df.empty:
+        task_report.summary = "No SSH authentication events in input files."
+
```

Comment:
there seems to be unnecessary white lines in different places in different files, is this maybe an editor setting you have?